### PR TITLE
Update operations docu

### PR DESCRIPTION
**Pull request by NHingerl**
**Description**

Changes proposed in this pull request:

- integrate CLI docu with main Kyma docs
- review existing operations guides
- review config guides
- review troubleshooting guides
- preliminary, to enable netlify preview: rename files and remove subfolders

**Related issue(s)**

### CODE REVIEW

File: docs/basic-tasks/00-task-install-kyma-CLI.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Install Kyma CLI
+type: Tasks
+---
+
+<!-- Installing the Kyma CLI is a "basic task"; users need the CLI to get started -->
```
Comment (klaudiagrz) [inline docs/basic-tasks/00-task-install-kyma-CLI.md:6]: Yes, fully agree 👍

File: docs/basic-tasks/00-task-provision-cluster.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+---
+title: Provision a cluster
+type: Tasks
+---
+<!-- this is surely mentioned in Basic Tasks/Get Started -->
```
Comment (klaudiagrz) [inline docs/basic-tasks/00-task-provision-cluster.md:5]: 👍

File: docs/operation-guides/config-09-apigateway-security_options.md [hunk 3]
```diff
@@ -1,22 +1,26 @@
 ---
-title: Available security options
-type: Details
+title: Configure Authorization (OAuth2, JWT)
+type: Configuration
```
Comment (klaudiagrz) [inline docs/operation-guides/config-09-apigateway-security_options.md:3]: I'm wondering whether it should be security that KK mentioned today 🤔

**Review by klaudiagrz — APPROVED** [review]

File: docs/operation-guides/config-09-apigateway-security_options.md (hunk 3 above)
Comment (NHingerl) [reply docs/operation-guides/config-09-apigateway-security_options.md:3]: I think so!

**Review by klaudiagrz — APPROVED** [review]

**Pull request merged**